raft/src/test/java/org/apache/kafka/raft/LeaderStateTest.java

```diff
@@ -224,7 +314,9 @@ public void testGetVoterStates() {
             mkEntry(localId, leaderEndOffset),
             mkEntry(node1, leaderStartOffset),
             mkEntry(node2, leaderEndOffset)
-        ), state.getVoterEndOffsets());
+        ), state.quorumResponseVoterStates(0)
+                .stream()
```

Comment:
nit: indentation looks kind of strange here. Probably should be 4 spaces? Also below on lines 344 and 368.